### DISCUSSION TITLE
how to set width on kas::column!

### DISCUSSION THREAD

**Discussion by mclellanisaac** [kas-gui/kas#435] [Q&A]
curious on how I could set a width on a column

**Comment by dhardy** [comment]
You don't directly.

Each widget has a size requirement. If you want to make a widget larger (one *inside* the column), the easiest way is [`with_min_size_px`](https://docs.rs/kas/latest/kas/widgets/trait.AdaptWidget.html#method.with_min_size_px) / `_em`.
